Week-1/Web-Page/index.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
```

Comment:
you need to upload the writing assignment by using creating readme.md file types . pdf and docx files are not readable ?

Reply:
i was unaware of that , i ll take care from the next time,Sorry.